test/extended/deployments/deployments.go

```diff
@@ -1089,7 +1089,7 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
 			})
 
 			g.By("deleting owned RCs when deleted", func() {
-				err = oc.Clientset().AppsV1().DeploymentConfigs(namespace).Delete(dcName, &metav1.DeleteOptions{})
+				err = oc.AppsClient().Apps().DeploymentConfigs(namespace).Delete(dcName, &metav1.DeleteOptions{})
```

Comment:
@deads2k @mfojtik why moving this to internal?

Reply:
> @deads2k @mfojtik why moving this to internal?

It happened to be what was available here.  I have no particular preference at the moment.